### PR TITLE
Xui cve fix

### PR DESCRIPTION
**Pull request by ernestman28**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by paul-graham — APPROVED** [review]
All looks fine to me, although I'm not entirely sure I approve of the added space in the `README.md`. 🤔

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ up through AKS deployed through Flux to AAT, ITHC and Prod, the application will
 The developer needs to set these up locally, so that the developer can see any issues early in
 the development process, and not when the application is placed up onto the higher AKS environments.
 
-To setup the secrets locally do the following: 
+To setup the secrets locally do the following:
```
Comment (paulhowes-HMCTS) [inline README.md:77]: Little unnecessary space at the end of the line, but I can forgive you for this

**Review by paulhowes-HMCTS — APPROVED** [review]
Just a tiny finicky comment but approved!

**Comment by ernestman28** [comment]
> All looks fine to me, although I'm not entirely sure I approve of the added space in the `README.md`. 🤔

thats was me just getting a initial PR build so ensure infra was stable before any changes

**Pull request merged**